### ISSUE TITLE
Set upper and lower limits in Line Chart

### ISSUE THREAD

**Issue by vigneshtg** [AnyChart/AnyChart-Android#201]
Hello,
I'm using Line chart and is it possible  to add upper and lower limits for y-axis?
I mean to set min and max value  for Y-Axis.

Thank you.

**Comment by Shestac92** [comment]
@vigneshtg 
Yes, you can apply minimum and maximum values to the chart yScale. For details, check [the gist](https://gist.github.com/Shestac92/c6392c6ad6259709295884aaf5eaeeed).

**Comment by vigneshtg** [comment]
@Shestac92 Thank you for the help.
